.github/workflows/appengine-deploy.yml

```diff
@@ -52,3 +52,15 @@ jobs:
       - id: test
         name: Test the deloyed URL
         run: curl --fail "${{ steps.deploy.outputs.url }}"
+        
+      - name: Happo
+        working-directory: web
+        run: npm run happo-ci-github-actions
+        env:
+          PREVIOUS_SHA: 490c6fa004074f6cf23649f5336f1e9004f4574d
```

Comment:
I'm a bit confused about this commit hash here... Shouldn't the picture of the current hash be compared with the previous one automatically?

Reply:
I am hardcoding the hash to be a hash of the "golden images". If I left it out, it would be comparing it to the previous commit in the same branch, which is not going to be very helpful since we want to compare our screenshots of the app in the branch to the screenshots of the app from the main branch.